### PR TITLE
chore(analytics, advanced_analytics): 11670 - decouple disaster links from analytical panel

### PR DESCRIPTION
**Pull request by IlyaIzr**
Links moved from analytics to active event card according to new design
image

### CODE REVIEW

File: src/features/events_list/components/EventCard/EventCard.module.css [hunk 1]
```diff
@@ -29,3 +30,20 @@
...
+  flex-direction: column;
+  gap: var(--unit);
+  font-family: var(--font-family);
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.markdown {
```
Comment (propakov) [inline src/features/events_list/components/EventCard/EventCard.module.css:41]: you've added it there, but haven't removed it from AnalyticsData.module.css

File: src/features/events_list/components/EventCard/EventCard.tsx [hunk 2]
```diff
@@ -55,6 +61,22 @@ export function EventCard({
         loss={event.loss}
       />
 
+      {isActive && externalUrls?.length ? (
+        <div>
```
Comment (propakov) [inline src/features/events_list/components/EventCard/EventCard.tsx]: it seems like this level of div is redundant 
I've removed it and nothing has changed

File: src/features/events_list/components/EventCard/EventCard.module.css [hunk 3]
```diff
@@ -29,3 +30,20 @@
...
 }
+
+.linkContainer {
+  display: flex;
+  flex-direction: column;
+  gap: var(--unit);
+  font-family: var(--font-family);
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
```
Comment (propakov) [inline src/features/events_list/components/EventCard/EventCard.module.css]: --font-xs

Comment (propakov) [reply src/features/events_list/components/EventCard/EventCard.module.css]: oh, we also have k-font-caption class in typography that repeats all text styles from here
idea was to have 2 classes .linkContainer {
  display: flex;
  flex-direction: column;
  gap: var(--unit);
}  + k-font-caption 
just to reuse existing styles
but not sure if it's a right idea to use it out of component  🤔 
we have only one such usage in our codebase

**Review by propakov — APPROVED** [review]
LGTM
Found some possible moments to improve, pls consider

File: src/features/events_list/components/EventCard/EventCard.module.css (hunk 3 above)
Comment (IlyaIzr) [reply src/features/events_list/components/EventCard/EventCard.module.css]: it works but yes it's just string out of context, feels like smth everyone would ask then 'where did it came from?'
image

**Review by Akiyamka — APPROVED** [review]

**Review by Komzpa — APPROVED** [review]
Thanks for raising the coverage :)

**Pull request merged**